test/hotspot/jtreg/vmTestbase/nsk/jvmti/GetThreadState/thrstat002/thrstat002.cpp

```diff
@@ -48,6 +48,104 @@ static jint state[] = {
     JVMTI_THREAD_STATE_IN_OBJECT_WAIT
 };
 
+#define DBG 1
```

Comment:
I don't think that DBG really needs to be disabled, just print info anytime when test fails

Reply:
I used these ifdef's to easily identify the debugging/utility fragments in a case we want to isolate it in testing library. It does not harm anyway.